lib/mi_bridges/driver/go_back_to_application_page.rb

```diff
@@ -6,12 +6,18 @@ class GoBackToApplicationPage < BasePage
       def setup; end
 
       def fill_in_required_fields
-        choose " Go back to where you stopped filling out the application."
+        choose_go_back_to_where_left_off
```

Comment:
yaaaas

Reply:
interestingly, choosing the label did not work but choosing that random ID does 🤦‍♂️